docs/roadmap.md

```diff
@@ -44,17 +45,22 @@ Instead of asking an administrator to change Argo CD settings end users can perf
 
 ## v2.3 and beyond
 
-### Application Details Page Usability
+### Input Forms UI Refresh
+
+Improved design of the input forms in Argo CD Web UI: https://www.figma.com/file/IIlsFqqmM5UhqMVul9fQNq/Argo-CD?node-id=0%3A1
+
+### Merge ApplicationSet controller into Argo CD
```

Comment:
👍 

Minimally, move the repo. Cautiously, move the controller into the same process.